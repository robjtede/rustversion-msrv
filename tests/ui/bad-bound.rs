#[rustversion_msrv::since(stable)]
struct S;

#[rustversion_msrv::any(since(stable))]
struct S;

fn main() {}
