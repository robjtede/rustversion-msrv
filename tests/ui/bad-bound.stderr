error: expected rustc release number like 1.31, or nightly date like 2024-05-12
 --> tests/ui/bad-bound.rs:1:27
  |
1 | #[rustversion_msrv::since(stable)]
  |                           ^^^^^^

error: expected rustc release number like 1.31, or nightly date like 2024-05-12
 --> tests/ui/bad-bound.rs:4:31
  |
4 | #[rustversion_msrv::any(since(stable))]
  |                               ^^^^^^
